Documents/8-Kigeme-Rwanda-Use-Case.md

```diff
@@ -0,0 +1,239 @@
+![](media/f02e74c3599518984984f5a9dd1d0349.png)
+
+**Humanitarian Engineering for Energy for Displacement (HEED)**
+
+##### Funded by the ESPRC Global Challenges Research Fund
+
+### Renewable Energy Recommendations Tool (RERT) 
+
+**Use Case:** Kigeme Camp Rwanda - Cooking Solutions Evaluation
```

Comment:
This should be the heading. Recommend using one `#` so that it comes up as a new section